src/components/analytics/AnalyticsCharts.jsx

```diff
@@ -1,8 +1,7 @@
-import React, { useState, useEffect } from 'react';
+import React, { useState, useRef, useEffect } from 'react';
 import PropTypes from 'prop-types';
 import { logError } from '@edx/frontend-platform/logging';
-
-import TableauReport from 'tableau-react-embed';
+import url from 'url';
```

Comment:
Is this library built-in available in react or do we need to install it manually? Asking it because if we have to install it manually then we should add it in the Package.json file as well.

Reply:
This is available as part of node